lib/hydra/pcdm/services/collection/get_related_objects.rb

```diff
@@ -10,8 +10,7 @@ class GetRelatedObjectsFromCollection
 
     def self.call( parent_collection )
       raise ArgumentError, "parent_collection must be a pcdm collection" unless Hydra::PCDM.collection? parent_collection
-
-      parent_collection.related_objects
+      parent_collection.related_objects.to_a
```

Comment:
Do we need to call to_a here?  Why?  to_a can be less efficient if they only want the first element of the association.